Bottom: 8377f6ea8779fa005e5b6cd106b99f93245d305c
Top:    ade089a79c15801f412bd0c14584a3de061643dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 03:33:00 +0100

Refresh of shiny-3d-eye-candy

---

diff --git a/configure.ac b/configure.ac
index ba1afc7..31eec7d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -978,7 +978,7 @@ AM_CONDITIONAL(DEBUG_BUILD, test x$enable_debug = xyes)
 # ------------- Complete set of CFLAGS and LIBS -------------------
 
 CFLAGS="$CFLAGS $X_CFLAGS $DBUS_CFLAGS $GLIB_CFLAGS $GTK_CFLAGS $CAIRO_CFLAGS $GTKGLEXT_CFLAGS"
-LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS -lglut"
+LIBS="$LIBS $XM_LIBS $DBUS_LIBS $X_LIBS $GLIB_LIBS $GTK_LIBS $DMALLOC_LIBS $GD_LIBS $INTLLIBS $CAIRO_LIBS $GTKGLEXT_LIBS"
 
 
 # if we have gcc then add -Wall
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index de48e56..be82e4e 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2672,7 +2672,6 @@ ghid_parse_arguments (int *argc, char ***argv)
 
   gtk_init (argc, argv);
   gtk_gl_init(argc, argv);
-//  glutInit (argc, argv);
 
   gport = &ghid_port;
   fprintf (stderr, "gport set at %p\n", gport);
